build.cmd

```diff
@@ -242,12 +242,10 @@ set __RunArgs=-BuildOS=%__BuildOS% -BuildType=%__BuildType% -BuildArch=%__BuildA
 
 if %__EnforcePgo%==1 (
     if %__BuildArchArm%==1 (
-        echo Error: enforcepgo cannot be used with arm architecture
-        goto Usage
+        echo WARNING: enforcepgo does nothing with arm architecture
```

Comment:
Hmm, I think the warning text is fine. If some parsing picks it up somehow, can follow up. In the future maybe it can be turned back into a hard failure and we can configure the build better to pass it in the specific places it's needed.